src/actions/shop.actions.ts

```diff
@@ -0,0 +1,74 @@
+import { BrowserContext, expect, Page } from '@playwright/test';
+import ShopPage from '../pages/shop.pages';
+import BaseActions from './base/base.actions';
+import shopData from '../fixture/smoke/shop.fixture';
+
+class ShopActions extends BaseActions {
+  shopPage: ShopPage;
+
+  constructor(page: Page, context: BrowserContext) {
+    super(page, context);
+    this.shopPage = new ShopPage(page, context);
+  }
+
+  async blockAdverticement() {
+    await this.context.route('**/*', request => {
+      return request.request().url().includes('googlesyndication.com') ? request.abort() : request.continue();
+    });
+  }
+
+  async getPrices() {
+    const prices = await this.shopPage.prices();
+    const pricesDigits = await Promise.all(
+      prices.map(async el => Number((await el.innerText()).replace(/[^\d.-]/g, '')))
+    );
+    return pricesDigits;
+  }
+
+  async filterPrice(targetAmount: number) {
```

Comment:
I think this method doesn't work. Try to click on filter button and use left or right button in loop.
```typescript
async filterPrice(targetAmount: number) {
  for(let i = 0; i < targetAmount) {
    await page.keyboard.press('ArrowLeft');
  }
}
```